### PR TITLE
Smooth slide-up animation of Event Cards now occurs

### PR DESCRIPTION
**Pull request by dreadpool2**
32957146_1735272249841408_463897648114434048_n

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt [hunk 1]
```diff
@@ -18,6 +18,9 @@ class EventsFragment : Fragment() {
     private val eventsRecyclerAdapter: EventsRecyclerAdapter = EventsRecyclerAdapter()
     private val eventsViewModel by viewModel<EventsViewModel>()
     private lateinit var rootView: View
+    private val linearLayoutManager: LinearLayoutManager by lazy {
+        LinearLayoutManager(activity)
+    }
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: App will crash on back press

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: Updated the PR

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: This should be lateinit var

**Comment by dreadpool2** [comment]
Updated PR

**Pull request merged**